brainscore/benchmarks/__init__.py

```diff
@@ -218,8 +218,8 @@ def _evaluation_benchmark_pool():
             lambda benchmark_ctr=benchmark_ctr: benchmark_ctr())
 
     # Zhu2019
-    from .zhu2019 import Zhu2019RDM
-    pool['yuille.Zhu2019-rdm'] = LazyLoad(Zhu2019RDM)
+    from .zhu2019 import Zhu2019Accuracy
+    pool['yuille.Zhu2019-accuracy'] = LazyLoad(Zhu2019Accuracy)
```

Comment:
I suggest naming this "AccuracyMatch" and keeping the engineering one at "Accuracy"